### ISSUE TITLE
ASLLVD corrections - part 7; and still a problem with duplicate occurrences associated with a compound

### ISSUE THREAD

**Issue by cneidle** [DCS-LCSR/ASL-DAI#158]
Here are a few more corrections:

[asllvd-corrections-part7.xlsx](https://github.com/DCS-LCSR/ASL-DAI/files/3427367/asllvd-corrections-part7.xlsx)

Note that some of these involve compounds.  In particular, there are compounds where there is still a mismatch between the examples we see in the composite video and the occurrences that are displayed.

<img width="868" alt="Screen Shot 2019-07-24 at 10 28 25 AM" src="https://user-images.githubusercontent.com/13629122/61802285-31c80000-adfe-11e9-9fa5-563b34286bfb.png">

<img width="627" alt="Screen Shot 2019-07-24 at 10 28 18 AM" src="https://user-images.githubusercontent.com/13629122/61802300-38567780-adfe-11e9-90c0-cef8e61a868f.png">

<img width="779" alt="Screen Shot 2019-07-24 at 10 36 57 AM" src="https://user-images.githubusercontent.com/13629122/61802900-3ccf6000-adff-11e9-9eec-3116dcaa41d6.png">

<img width="373" alt="Screen Shot 2019-07-24 at 10 36 50 AM" src="https://user-images.githubusercontent.com/13629122/61802901-3d67f680-adff-11e9-9d60-db442ff1753c.png">

So, I put a ? in the ID column where those glosses are in need of being modified.  

And as previously observed, there are cases where the parts of the compound are not listed at all, so I can't indicate the corresponding changes.

Basically MILLION and BILLION need to be changed to MILLION/BILLION in both cases everywhere they occur with BILLION --> MILLION/BILLION+, and MILLION_2 and BILLION_2 need to be changed to MILLION/BILLION_2.  This is true for all the separate signs, and for all the compounds in which they participate (including the listing of the parts of those compounds, obviously).  

This is all very confusing.  Perhaps it makes sense to straighten out the compound displays first, and then worry about changing the glosses that are involved in the compounds...  Perhaps we can discuss how best to proceed on these ??  I can identify the necessary changes again once this is sorted out properly... 

Thanks!

**Comment by cneidle** [comment]
THERE ARE STILL PROBLEMS with the following:


1.

<img width="1273" alt="Screen Shot 2019-07-26 at 8 37 52 AM" src="https://user-images.githubusercontent.com/13629122/61952222-4c76b200-af81-11e9-8cfb-79eeef64f45a.png">



2. 

<img width="657" alt="Screen Shot 2019-07-26 at 8 35 16 AM" src="https://user-images.githubusercontent.com/13629122/61952223-4c76b200-af81-11e9-9745-c8354a44c243.png">

<img width="869" alt="Screen Shot 2019-07-26 at 8 35 00 AM" src="https://user-images.githubusercontent.com/13629122/61952224-4c76b200-af81-11e9-8989-ebebbe0d4de1.png">


Here is an updated version of the Part 7 corrections, with yellow highlighting for the problem cases...   THANKS.

[asllvd-corrections-part7-revised.xlsx](https://github.com/DCS-LCSR/ASL-DAI/files/3435763/asllvd-corrections-part7-revised.xlsx)

**Comment by kwasiopoku** [comment]
Done

**Comment by cneidle** [comment]
THanks so much.  THe labels for occurrences have been changed, but not the labels for the titles of each of these display pages, i.e., the listings in the index, and the groupings need to be adjusted according to the changes.

We should end up with:

MILLION/BILLION
     MILLION/BILLION
     MILLION/BILLION* (MILLION/BILLION+AGENT)
   * MILLION/BILLION (ONE+MILLION/BILLION)
     MILLION/BILLION_2
     MILLION/BILLION_2* (MILLION/BILLION_2+AGENT)
   
MILLION/BILLION+AGENT

MILLION/BILLION_2+AGENT

ONE+MILLION/BILLION

ONE
     ONE*  (ONE+MILLION/BILLION)


Right now we have:  

Title and index listing should be MILLION/BILLION [primary entry: MILLION/BILLION]

<img width="776" alt="Screen Shot 2019-08-01 at 6 04 40 AM" src="https://user-images.githubusercontent.com/13629122/62284921-a284a300-b422-11e9-8ee1-f56abc9b35f7.png">

Title and index listing should be MILLION/BILLION (and should be merged with the above)

<img width="797" alt="Screen Shot 2019-08-01 at 5 59 38 AM" src="https://user-images.githubusercontent.com/13629122/62284929-a31d3980-b422-11e9-8738-22fe65c6455f.png">

_____

Title and index listing should be MILLION/BILLION_2  [primary entry: MILLION/BILLION]

<img width="787" alt="Screen Shot 2019-08-01 at 6 04 50 AM" src="https://user-images.githubusercontent.com/13629122/62284920-a284a300-b422-11e9-96a0-d11bc1502370.png">

_____

Title and index listing should be MILLION/BILLION_2+AGENT

<img width="794" alt="Screen Shot 2019-08-01 at 6 00 51 AM" src="https://user-images.githubusercontent.com/13629122/62284923-a31d3980-b422-11e9-9450-731f4f02bce7.png">

_____

Title and index listing should be MILLION/BILLION_2* (MILLION/BILLION_2+AGENT)

<img width="782" alt="Screen Shot 2019-08-01 at 6 00 40 AM" src="https://user-images.githubusercontent.com/13629122/62284924-a31d3980-b422-11e9-805f-0b099657601d.png">

_____

Title and index listing should be MILLION/BILLION+AGENT

<img width="794" alt="Screen Shot 2019-08-01 at 6 00 29 AM" src="https://user-images.githubusercontent.com/13629122/62284925-a31d3980-b422-11e9-9d39-c01ead20da05.png">

Title and index listing should be MILLION/BILLION* (MILLION/BILLION+AGENT)

<img width="798" alt="Screen Shot 2019-08-01 at 6 00 12 AM" src="https://user-images.githubusercontent.com/13629122/62284926-a31d3980-b422-11e9-83d7-5cd2b7853345.png">

_____

Title and index listing should be *MILLION/BILLION (ONE+MILLION/BILLION)

<img width="797" alt="Screen Shot 2019-08-01 at 5 59 55 AM" src="https://user-images.githubusercontent.com/13629122/62284927-a31d3980-b422-11e9-9564-92e57df25d64.png">

_____

Title and index listing should be ONE* (ONE+MILLION/BILLION)

<img width="786" alt="Screen Shot 2019-08-01 at 6 23 16 AM" src="https://user-images.githubusercontent.com/13629122/62286061-f8f2e100-b424-11e9-933e-54c19ab76efd.png">


Also the sign highlighted in blue that was to have been modified seems to have disappeared:

<img width="1282" alt="Screen Shot 2019-08-01 at 6 03 47 AM" src="https://user-images.githubusercontent.com/13629122/62284922-a284a300-b422-11e9-8f93-507ff6baaefa.png">

**Comment by kwasiopoku** [comment]
Done

**Comment by cneidle** [comment]
This occurrence of OFTEN has not reappeared.  There should be 3 occurrences (see composite video) but only two are listed.  The one that was to have been modified is missing entirely.

**Comment by cneidle** [comment]
Hi Augustine - I was going to check this out, but the last problem noted with OFTEN has not been resolved.  Just FYI.  Thanks!

**Comment by cneidle** [comment]
There are still major issues with MILLION and BILLION, which should all have been converted to MILLION/BILLION (with some number of +'s).  It looks like we have both the "before" versions now, as well as the "after" versions, as duplicates, for compounds that originally contained either MILLION or BILLION.  Note that MILLION_2 was also to have been changed to MILLION/BILLION_2.   Thanks!

<img width="328" alt="Screen Shot 2019-09-02 at 12 29 20 PM" src="https://user-images.githubusercontent.com/13629122/64126908-94a5a300-cd7d-11e9-80ce-4301c247a313.png">

**Comment by kwasiopoku** [comment]
The issue with OFTEN and the MILLION/BILLION have been resolved.

**Comment by cneidle** [comment]
OFTEN looks good.

And the MILLION/BILLION stuff seems *almost* right.

The one fix that is now needed, I believe, is that the two compounds shown below should be separate main entries...  to be consistent with how we are dealing with other compounds.

MILLION/BILLION+AGENT
MILLION/BILLION_2+AGENT

We haven't been grouping together compounds based on their parts being under the same main entry.  And although we *could* do that, it would greatly complicate things... 

<img width="258" alt="Screen Shot 2019-09-02 at 3 23 52 PM" src="https://user-images.githubusercontent.com/13629122/64131332-e35f3700-cd95-11e9-9004-8e00d46c0316.png">

**Comment by kwasiopoku** [comment]
Fixed

**Comment by cneidle** [comment]
No - this is still not right.  Sorry.

MILLION/BILLION_2 should be (as it was) an entry/variant under MILLION/BILLION:

MILLION/BILLION
      MILLION/BILLION_2
      MILLION/BILLION_2* (MILLION/BILLION_2+AGENT)

However, the following 2 compounds should be distinct main entries:

MILLION/BILLION+AGENT
MILLION/BILLION_2+AGENT

ALSO: what you have as MILLION_2/BILLION needs to be MILLION/BILLION_2

thanks

<img width="313" alt="Screen Shot 2019-09-02 at 9 11 43 PM" src="https://user-images.githubusercontent.com/13629122/64137526-becf8300-cdc6-11e9-89ef-913c0b5cef07.png">

**Comment by kwasiopoku** [comment]
This has been updated
